### PR TITLE
Update Notebooks docs

### PR DESCRIPTION
**Pull request by tourunen**
Owners can now set session memory and lifetime. Clarify custom image
example. Remove beta reference as R5 is production now. Add explanation
for workspace memory limit.

### CODE REVIEW

File: docs/cloud/csc_notebooks/guide_for_teachers.md [hunk 1]
```diff
@@ -34,15 +34,23 @@ through `Application wizard` or `Application form`-buttons.
...
 **Container image** 
 * If using existing image, then pre-filled based on chosen Application template.
-* If using own custom image, then path to the Docker image. If using Rahti, then: `docker-registry.rahti.csc.fi/<yourprojectname>/<yourimagename>:<tag>` .
+* If using own custom image, then URL of the Docker image. In case of Rahti, this would be something like
+  `docker-registry.rahti.csc.fi/<yourprojectname>/<yourimagename>:<tag>`.
+
+**Session lifetime** The maximum lifetime for a single session. Sessions are deleted when they expire, thus making room 
+for other users.
+
+**Session memory** RAM memory reserved for each session. Workspaces have a total limit for total memory for concurrent
```
Comment (katrite) [inline docs/cloud/csc_notebooks/guide_for_teachers.md:52]: a limit for total memory

File: docs/cloud/csc_notebooks/guide_for_teachers.md [hunk 2]
```diff
@@ -34,15 +34,23 @@ through `Application wizard` or `Application form`-buttons.
...
+sessions, choosing a higher value here will affect the maximum number of concurrent sessions.
```
Comment (katrite) [inline docs/cloud/csc_notebooks/guide_for_teachers.md:53]: ...for concurrent sessions. Choosing a higher value here will affect the maximum number of concurrent sessions.

**Review by katrite — COMMENTED** [review]
Looks good! :) Added just few style -comments.

**Review by katrite — APPROVED** [review]
Looks good! :)

**Review by katrite — APPROVED** [review]

**Review by katrite — APPROVED** [review]

**Review by attesillanpaa — APPROVED** [review]

**Pull request merged**